utilities/unittests/test_artifactory.py

```
@@ -332,25 +338,26 @@ def test_get_artifactory_secret_returns_existing_if_exists(self, mock_base64_enc
 
-            result = get_artifactory_secret(namespace="test-namespace")
+            secret, created = get_artifactory_secret(namespace="test-namespace", client=MagicMock())
 
             # Verify deploy was NOT called
             mock_secret_instance.deploy.assert_not_called()
-            assert result == mock_secret_instance
+            assert secret == mock_secret_instance
+            assert created is False
 
     @patch("utilities.artifactory.Secret")
     def test_get_artifactory_secret_raises_key_error_if_user_not_set(self, mock_secret_class):
```

Ruff (0.16.0)

[warning] Unused method argument: `mock_secret_class`

(ARG002)